regression-tests/run-tests.sh

```diff
@@ -39,7 +39,7 @@ check_file () {
         failure=1
     else
         # Compare the content with the reference value checked in git
-        diff_output=$(git diff --ignore-cr-at-eol -- "$file")
+        diff_output=$(git diff --ignore-cr-at-eol -I"\#define CPP2\_.*\_STD" -- "$file")
```

Comment:
Should this also be added to line 35?

Reply:
No, IIUC, this branch is to handle the case where the test result expected output does not exist in Git, so the `git diff` here is being used to add the new content to a larger `.diff` file. CC @jarzec